logout.php

```diff
@@ -2,12 +2,25 @@
 require_once __DIR__ . '/vendor/autoload.php';
 
 session_start();
+Dotenv::load(__DIR__);
 
 $auth_factory = new \Aura\Auth\AuthFactory($_COOKIE);
 $auth = $auth_factory->newInstance();
 
-$logout_service = $auth_factory->newLogoutService($pdo_adapter);
```

Comment:
could you please send a PR with `$pdo_adapter` being null .

So `$logout_service = $auth_factory->newLogoutService(NULL);` . The reason is for logout the adapter is not doing anything. So even a NullAdapter works. See https://github.com/auraphp/Aura.Auth/blob/8a2f6412f2e27b6ce174acd74390d7b0cf3970ce/src/AuthFactory.php#L115-L121